backend/eventd/eventd.go

```diff
@@ -270,26 +273,37 @@ func (e *Eventd) dead(key string, prev liveness.State, leader bool) {
 	// here, filter by check, and update all events involved in the round robin
 	if entity == "" {
 		lager.Error("round robin check TTL not supported")
-		return
+		return false
+	}
+
+	ctx := store.NamespaceContext(context.Background(), namespace)
+
+	// The entity has been deleted, and so there is no reason to track check
+	// TTL for it anymore.
+	if ent, err := e.store.GetEntityByName(ctx, entity); err == nil && ent == nil {
+		fmt.Println("LOL NO ENTITY:", entity)
```

Comment:
I'm assuming this `Println()` was only meant for testing? :)

Reply:
It’s a better debug print statement than I usually go for 😂